compiler/bin-wasmoo_util/dune

```diff
@@ -0,0 +1,17 @@
+(executable
+ (name wasmoo_util)
```

Comment:
Have you consider having subcommands inside wasm_of_ocaml instead ?

Reply:
We have the issue that `wasmoo_util link` is used to build the runtime which is then included in `wasm_of_ocaml`.
But maybe I can have an internal binary to do that and also expose the same functionality as a wasm_of_ocaml subcommand.

Reply:
Shouldn't the runtime be linked later ? If we want to have it conditional on flags such as use-js-strings ?

Reply:
My current plan is to have a set of precompiled runtimes for the most common options, and to compile runtimes on the fly otherwise.
It's a bit slow to compile and link the runtime. That's especially noticeable when we have a large number of executables, like with the Js_of_ocaml test suite.
But maybe we could share the runtimes produced by `wasm_of_ocaml build-runtime` between executables, using a similar trick in dune as for ppxs?

Reply:
Sounds good